packages/frames.js/src/middleware/images-worker/handler.tsx

```diff
@@ -1,10 +1,7 @@
 import { ImageResponse } from "@vercel/og";
-import type { ReactElement } from "react";
-import React from "react";
```

Comment:
Getting `ReferenceError: React is not defined` because of this

Reply:
where are you getting this error?

Reply:
tests are passing just fine also building the frames.js.

Reply:
open the images worker example http://localhost:3010/?url=http://localhost:3000/examples/images-worker

![Screenshot 2024-06-04 at 10 31 48](https://github.com/framesjs/frames.js/assets/5469870/b67ed0a6-a39e-4012-bc76-5ce15b000e9a)

Reply:
hmm interesting

Reply:
fixed